opentelemetry-sdk/src/trace/sampler/jaeger_remote/mod.rs

```diff
@@ -1,4 +1,5 @@
 mod rate_limit;
+#[allow(dead_code)]
```

Comment:
seems new beta rust toolchain (1.81.x) is more strick to unused code. It seems `remote.rs` is generated from [proto ](https://github.com/jaegertracing/jaeger-idl/tree/main/proto) file so not adding any code there. Instead making whole generated to allow dead-code.